exporters/failure/collector_pagerduty.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
+#
+# Copyright Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+#
+
+import logging
+
+import requests
+from attrs import define, field
+
+from failure.collector_base import AbstractFailureCollector, TrackerIssue
+from pelorus.config import env_var_names, env_vars
+from pelorus.config.converters import comma_or_whitespace_separated
+from pelorus.config.log import REDACT, log
+from pelorus.errors import FailureProviderAuthenticationError
+from pelorus.timeutil import parse_assuming_utc, second_precision
+from pelorus.utils import TokenAuth, set_up_requests_session
+
+_DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
```

Comment:
My datetime format is correct? I am concerned with the `Z` (it comes as `2023-02-08T17:29:24Z`) and if I used the correct functions to handle it

Reply:
Closing this as the datatime format is correct here.